src/app/fdctl/configure/workspace.c

```diff
@@ -139,6 +146,20 @@ static void buf( void * pod, char * fmt, void * value, ulong sz, ... ) {
     FD_LOG_ERR(( "failed to initialize workspace" ));
 }
 
+
+
+static inline uchar *
+read_key( char const * key_path,
+          uchar      * key       ) {
+  FILE * key_file = fopen( key_path, "r" );
+  if( FD_UNLIKELY( !key_file ) ) FD_LOG_ERR(( "Opening key file (%s) failed", key_path ));
```

Comment:
(%i-%s), errno, strerror( errno )

Reply:
Can you make the ENOENT case a special descriptive error message, I feel like it's going to happen frequently.

```C
if( FD_UNLIKELY( errno == ENOENT ) ) FD_LOG_ERR(( "The [consensus.identity_path] in your configuration expects a keyfile at %s but there is no such file. Either update the configuration file to point to your validator identity keypair, or generate a new validator identity key by running `fdctl keygen`", key_path ));
```